CONTRIBUTING.md

```diff
@@ -11,3 +18,56 @@ I am truly grateful for all the support developing VSTeam Quickstart Scripts. It
 ### Housekeeping
 
 The VSTeam module runs on macOS, Windows and Linux. Therefore, **casing is very important**.  If your script only works on one platform please describe in the Readme.md file. Also note if this only support a specific version of TFS or requires a minimum API version.
+
+## Contribution Guide
+
+To make sure your scripts are easy to use, please follow these guidelines.
+
+## Files, folders and naming conventions
+
+1. Every script and its associated files must be contained in its own **folder**. Name this folder something that describes what your script does. Usually this naming pattern looks like **level-resource-action** (e.g. 101-git-stale-branches)
+
++ **Required** - Numbering should start at 101. 100 is reserved for things that need to be at the top.
```

Comment:
I learned something. Not sure though why sometimes ordering is enforced. in ARM quickstart I have seen also 201 examples. Do you know?

Reply:
I don't know.